features/admin.users.v1/constants/user-management-constants.ts

```diff
@@ -87,6 +87,7 @@ export class UserManagementConstants {
         .set("USER_ROLES", "users.edit.roles")
         .set("USER_SHARED_PROFILES", "users.updateSharedProfiles")
         .set("USER_IMPERSONATION", "users.user.impersonation")
+        .set("HIDE_READ_ONLY_ATTRIBUTES_WHEN_EMPTY", "profile.hideReadonlyAttributesWhenEmpty")
```

Comment:
Shall we add the key to the `UserFeatureDictionaryKeys` and reuse? It will improve the reusability and make it fail-proof. Ex: line 91